Sources/Extensions/Foundation/ArrayExtensions.swift

```diff
@@ -34,8 +35,10 @@ public extension Array where Element: FloatingPoint {
 	///
 	/// - Returns: average of the array's elements.
 	public func average() -> Element {
-		// http://stackoverflow.com/questions/28288148/making-my-function-calculate-average-of-array-swift
-		return isEmpty ? 0 : reduce(0, +) / Element(count)
+        guard isEmpty == false else { return 0 }
```

Comment:
@SD10, I see, that makes sense as well 🤔
I'm cool with both styles, feel free to do whatever you feel is more readable 👍